package.json

```diff
@@ -31,6 +31,7 @@
     "react": "^15.4.2",
     "react-dom": "^15.4.2",
     "request": "^2.79.0",
+    "unirest": "^0.5.1",
     "webpack": "^1.14.0",
     "webpack-dev-server": "^1.16.2"
   },
```

Comment:
Should we add a script to run the sever in package.json (now that the server is functional)?